spec.md

```diff
@@ -690,6 +695,7 @@ Procedure:
 28. return spk
 
 How a signature is to be encoded is not covered by this document. (TODO perhaps add some additional information in the appendix)
+(FIXME: Encoding out of scope OK, but wire format is required for test vectors. Unless this document is not normative.)
```

Comment:
I think I agree with your comment here but could you clarify what you mean by "wire format" specifically and what you would take as the definition of encoding format?

Reply:
As an implementer I would expect qualifiers such as "32 octets" or "variable length" (with a length field) for the members of the tuple and then a normative ordering of them (for example by having a wire format graphic or a concatenation of the octet strings).
The point is that when I eventually code my structs, I need to know the size of its members and their location.
Otherwise I do not even know how to parse a test vector (if provided).
This unclarity may have to do with #8 which makes normative statements more difficult to incorporate.

Reply:
Yeah I think this is covered by issue #2 and we should aim to review it on the next working group call to provide better clarity around the direction the WG would like to take this